data/ships.txt

```diff
@@ -1188,8 +1188,8 @@ ship "Firebird"
 	attributes
 		category "Medium Warship"
 		"cost" 3700000
-		"shields" 6400
-		"hull" 2800
+		"shields" 5800
+		"hull" 3400
```

Comment:
this does mean the Firebird will be disabled a little bit faster (total hp was 9200, stays 9200, previously disabled at 560, now 680), but shouldn't matter that much at all